Duplicate Headers From Include Directories

This issue highlights a problem in which the headers from include directories are listed twice to be formatted, as evidenced in this log snippet:

```
2/3 Test #2: Testing include directories formatting ...***Failed    1.53 sec
-- Getting the original source file hashes
-- Copying the dirty source files
-- Configuring sample project
-- The CXX compiler identification is AppleClang 13.1.6.13160021
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Configuring incomplete, errors occurred!
CMake Error at /Users/threeal/Workspaces/cmake-projects/FixFormat.cmake/test/FixFormatTest.cmake:42 (message):
  Failed to configure sample project: CMake Error at
  /Users/threeal/Workspaces/cmake-projects/FixFormat.cmake/cmake/FixFormat.cmake:35
  (message):

    FILES:
    /Users/threeal/Workspaces/cmake-projects/FixFormat.cmake/test/sample/src/fibonacci.cpp;/Users/threeal/Workspaces/cmake-projects/FixFormat.cmake/test/sample/src/is_odd.cpp;/Users/threeal/Workspaces/cmake-projects/FixFormat.cmake/test/sample/include/sample.hpp;/Users/threeal/Workspaces/cmake-projects/FixFormat.cmake/test/sample/include/sample/fibonacci.hpp;/Users/threeal/Workspaces/cmake-projects/FixFormat.cmake/test/sample/include/sample/is_odd.hpp;/Users/threeal/Workspaces/cmake-projects/FixFormat.cmake/test/sample/include/sample.hpp;/Users/threeal/Workspaces/cmake-projects/FixFormat.cmake/test/sample/include/sample/fibonacci.hpp;/Users/threeal/Workspaces/cmake-projects/FixFormat.cmake/test/sample/include/sample/is_odd.hpp

  Call Stack (most recent call first):

    CMakeLists.txt:27 (target_fix_format)

  



Call Stack (most recent call first):
  /Users/threeal/Workspaces/cmake-projects/FixFormat.cmake/test/FixFormatTest.cmake:75 (check_source_codes_format)
```

This issue seems to be caused by the following lines, in which we scan for header files in directories obtained from the `INCLUDE_DIRECTORIES` and `INTERFACE_INCLUDE_DIRECTORIES` properties:

https://github.com/threeal/FixFormat.cmake/blob/9614130a7b769e11fa5af6c1e3d08cbe66f5a003/cmake/FixFormat.cmake#L18-L20

After looking at the documentation, it seems that both may contain the same value. Since it's better to format only the header files of each target (instead of also the header files of the dependencies of each target), it's better to use only the `INCLUDE_DIRECTORIES` property for obtaining the include directories.

--- a/cmake/FixFormat.cmake
+++ b/cmake/FixFormat.cmake
@@ -16,15 +16,13 @@ function(target_fix_format TARGET)
   endif()
 
   # Append header files from include directories of the target to be formatted.
-  foreach(PROP INCLUDE_DIRECTORIES INTERFACE_INCLUDE_DIRECTORIES)
-    get_target_property(TARGET_INCLUDE_DIRS ${TARGET} ${PROP})
-    if(NOT "${TARGET_INCLUDE_DIRS}" STREQUAL TARGET_INCLUDE_DIRS-NOTFOUND)
-      foreach(INCLUDE_DIR ${TARGET_INCLUDE_DIRS})
-        file(GLOB_RECURSE HEADERS CONFIGURE_DEPENDS "${INCLUDE_DIR}/*")
-        list(APPEND FILES ${HEADERS})
-      endforeach()
-    endif()
-  endforeach()
+  get_target_property(TARGET_INCLUDE_DIRS ${TARGET} INCLUDE_DIRECTORIES)
+  if(NOT "${TARGET_INCLUDE_DIRS}" STREQUAL TARGET_INCLUDE_DIRS-NOTFOUND)
+    foreach(INCLUDE_DIR ${TARGET_INCLUDE_DIRS})
+      file(GLOB_RECURSE HEADERS CONFIGURE_DEPENDS "${INCLUDE_DIR}/*")
+      list(APPEND FILES ${HEADERS})
+    endforeach()
+  endif()
 
   # Append header files from file set of the target to be formatted.
   get_target_property(TARGET_HEADER_SET ${TARGET} HEADER_SET)

